keyboards/handwired/zergo/config.h

```diff
@@ -33,18 +33,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* key matrix pins */
 #define MATRIX_ROW_PINS { B1, D7, C3, D6, D5, D4 }
 #define MATRIX_COL_PINS { C7, C6, C5, C4, C2, C1, B7, D3, D2, B6, B5, B4, B3, B2 }
-#define UNUSED_PINS
+#define UNUSED_PINS { A0, A1, A2, A3, A4, A5, A6, A7, B0, C0, E0, E1, E4, E5, F0, F1, F2, F3, F4, F5, F6, F7 }
```

Comment:
Just a heads up, this value isn't actually used, and can be removed. 

Basically, it's only useful for reference.

Reply:
OK. I know